apps/server/src/modules/h5p-editor/uc/h5p.uc.ts

```diff
@@ -199,24 +208,40 @@ export class H5PEditorUc {
 		}
 	}
 
-	public async getH5pPlayer(currentUser: ICurrentUser, contentId: string): Promise<string> {
+	public async getH5pPlayer(currentUser: ICurrentUser, contentId: string): Promise<IPlayerModel> {
 		// TODO: await this.checkPermission...
 		const user = this.changeUserType(currentUser);
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-		const h5pPlayerHtml: string = await this.h5pPlayer.render(contentId, user);
-		return h5pPlayerHtml;
+		const playerModel: IPlayerModel = await this.h5pPlayer.render(contentId, user);
+		return playerModel;
 	}
 
-	public async getH5pEditor(currentUser: ICurrentUser, contentId: string, language: string): Promise<string> {
+	public async getEmptyH5pEditor(currentUser: ICurrentUser, language: string) {
 		// If contentId is undefined, a new H5P content will be created.
 		// TODO: await this.checkPermission...
 		const user = this.changeUserType(currentUser);
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-		const createdH5PEditor: string = await this.h5pEditor.render(contentId, language, user);
+		const createdH5PEditor: IEditorModel = await this.h5pEditor.render(undefined as unknown as string, language, user);
 
 		return createdH5PEditor;
 	}
 
+	public async getH5pEditor(currentUser: ICurrentUser, contentId: string, language: string) {
```

Comment:
```suggestion
	public async getH5pEditor(currentUser: ICurrentUser, contentId?: string, language?: string = 'de') {
```
or sort optionaly pros to back

Reply:
Not quite sure what you mean, contentId and language are not optional.
I see that I've left a comment that proclaims the opposite, but that one should be deleted.

(Language could be set to "de", but language selection is currently another ticket that is being worked on so I wouldn't change it in this PR)

Reply:
Ok, than change

`
export class GetH5PEditorParams {
	@ApiProperty()
	@IsMongoId()
	contentId!: string;
}
`